src/Components/Forms/src/EditContextDataAnnotationsExtensions.cs

```diff
@@ -16,98 +16,124 @@ namespace Microsoft.AspNetCore.Components.Forms
     /// </summary>
     public static class EditContextDataAnnotationsExtensions
     {
-        private static ConcurrentDictionary<(Type ModelType, string FieldName), PropertyInfo?> _propertyInfoCache
-            = new ConcurrentDictionary<(Type, string), PropertyInfo?>();
-
         /// <summary>
         /// Adds DataAnnotations validation support to the <see cref="EditContext"/>.
         /// </summary>
         /// <param name="editContext">The <see cref="EditContext"/>.</param>
+        [Obsolete("Use " + nameof(EnableDataAnnotationsValidation) + " instead.")]
```

Comment:
We should do a breaking change announcement for this.

Reply:
Agreed, will do so if/when this is merged.

Reply:
https://github.com/aspnet/Announcements/issues/458